lib/version-check.js

```diff
@@ -41,7 +41,7 @@ module.exports = function (gulp, gutil) {
 
                 //On success invoke callback function
                 if (typeof cb !== 'undefined') {
-                    cb();
+                    return cb();
```

Comment:
Ah ok. but what would be the result object in this case? Just an empty string perhaps?